### DISCUSSION TITLE
my program always crash

### DISCUSSION THREAD

**Discussion by LinGeLin** [uNetworking/uWebSockets#1021] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/24861783/78234293-8ed99100-7509-11ea-8fa5-85096ff667b9.png)

**Comment by LinGeLin** [comment]
how to avoid？

**Comment by ghost** [comment, marked as answer]
Why are you using old code? If I remember things you have to Node::addAsync() once at startup to have this threading support.
